ui/app/AppLayouts/Profile/views/SyncingView.qml

```diff
@@ -272,6 +271,30 @@ SettingsContentBase {
                     }
                 }
 
+                RowLayout {
+                    Layout.fillWidth: true
```

Comment:
Not needed, layout inside layout has fillWidth/fillHeight set to true by default. Ofc keep it if you just want to be explicit.